### ISSUE TITLE
question about the anastomoses

### ISSUE THREAD

**Issue by dpdclub** [phydev/angio#5]
Hi Maurício,

Sorry to bother you again. I read the paper, but not sure about the mechanism of forming the anastomoses. So the bifucation of the network mainly results from the generation of new tip agents or it could come from growing tip agents moving into an existing vessel.  I am trying to build two initial vessels and expect the newly generated vessel network from these two main vessels to merge, but it seems new vessels do not like to grow toward each other as the figure attached shows. Could you please give me some suggestions.

Thanks,

 
![1729212698981](https://github.com/user-attachments/assets/c956b9eb-3521-4bcc-ba17-e5159466a5e2)

**Comment by phydev** [comment]
Hi,

There is no mechanism for anastomosis in our code. Anastomoses happen due to the growth factors that direct more than one vessel to the same site. This happens more frequently with the deactivation by blood flow, because sources keep active for longer and can attract several vessels. 

I see that you added two main vessels as initial condition. I'm not sure if the blood flow computation works as expected with this condition. It might work if we set the boundary conditions in the input and output nodes for the whole box, but I don't remember how I implemented it.
